packages/calcite-components/src/components/stepper-item/stepper-item.tsx

```diff
@@ -123,6 +124,11 @@ export class StepperItem implements InteractiveComponent, LocalizedComponent, Lo
    */
   @Prop({ reflect: true }) scale: Scale = "m";
 
+  /**
+   * @internal
+   */
+  @Prop({ reflect: true }) responsiveMode = false;
```

Comment:
It seems odd to have an internal prop for being responsive. Shouldn't it always be in "responsive mode"? Maybe the prop name is just odd?

Reply:
I agree it is always in the responsive mode, the idea here was to change UI since the width has gone below the breakpoint but the name is confusing. Will come-up with a better name.